test/schema/conditional_workflow.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: beehive/v1
+kind: Workflow
+metadata:
+  name: beehive-deployment
+  labels:
+    app: example2
+spec:
+  strategy:
+    type: sequence
+  template:
+    metadata:
+      name: beehive-deployment
+      labels:
+        app: example
+        use-case: test
+  agents:
+    - agent1
+    - agent2
+    - agent3
+  exception: step3
+  steps:
+    - name: step1
```

Comment:
Is the first step the step executed first when the workflow is executed?

Reply:
How about adding `start` in the schema that specifies the start step name?

Reply:
I like this. However, is the start not implicit from the first step specified?

Reply:
Steps are executed in the order they are defined which is natural. No need for start but noted suggestion. Let see how far we can get...